src/mem/ruby/protocol/MOESI_AMD_Base-dir.sm

```diff
@@ -527,9 +533,9 @@ machine(MachineType:Directory, "AMD Baseline protocol")
         out_msg.ForwardRequestTime := tbe.ForwardRequestTime;
         out_msg.ProbeRequestStartTime := tbe.ProbeRequestStartTime;
         out_msg.OriginalResponder := tbe.LastSender;
-        if(tbe.atomicData){
-          out_msg.WTRequestor := tbe.WTRequestor;
-        }
+        out_msg.isGLCSet := tbe.isGLCSet;
+        out_msg.isSLCSet := tbe.isSLCSet;
+        out_msg.CURequestor := tbe.CURequestor;
```

Comment:
@BujSet: this integration with your patch looks wrong ... but also passes all tests.  Am I missing something?

Reply:
I went ahead and updated it in the latest patch, but would appreciate it @BujSet if you can confirm this looks right on your end.  Also rerunning all tests.